automation/src/test/scala/WorkspaceSpec.scala

```diff
@@ -60,6 +60,22 @@ class WorkspaceSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures[
         listPage.ui.hasWorkspace(billingProject, workspaceName) shouldBe false
       }
     }
+
+    "should be able to log out and log in as another user" in withWebDriver { implicit driver =>
```

Comment:
These test cases would be better in a different test suite, maybe something like `SignInSpec`.